composables/useUserOrder.ts

```diff
@@ -0,0 +1,46 @@
+import { computed } from "@vue/composition-api"
+import { buildSearchOrdersVars } from "./helpers/buildSearchOrdersVars"
+import { getOrdersQuery } from "@/lib/gql/queries"
+import { useState, useNuxtApp } from "#app"
+import type { Maybe, OrderCollection } from "@/server/types/GraphQL"
+
+export const useUserOrder = (referenceKey: string) => {
+  const nuxt = useNuxtApp()
+  const fetcher = nuxt.nuxt2Context.$gqlFetch
+
+  const result = useState<Maybe<OrderCollection>>(
+    `use-userOrders-result-${referenceKey}`,
+    () => null
+  )
+  const loading = useState<Boolean>(`use-userOrders-loading-${referenceKey}`, () => false)
+  const error = useState(`use-userOrders-error-${referenceKey}`, () => null)
+
+  const search = async (params: {
```

Comment:
Please rename "search" to "getOrders" as functionality inclined with get order details, not search.

Reply:
Resolved